models/prices/avalanche_c/prices_avalanche_c_tokens.sql

```diff
@@ -17,6 +17,7 @@ FROM
     VALUES
 
     ("aave-new","avalanche_c","AAVE.e","0x63a72806098bd3d9520cc43356dd78afe5d386d9",18),
+    ("glcr-glacier","avalanche_c","GLCR","0x3712871408a829C5cd4e86DA1f4CE727eFCD28F6",18),
```

Comment:
please note this id value isn't found on the API: https://api.coinpaprika.com/v1/coins/glcr-glacier

Reply:
could it be a different value?

Reply:
sorry, got a bit ahead of my self. didn't realize the coincaprika listing was a prerequisite.

I'll check back post listing (apparently about a month+)

Reply:
okay, will close PR for now, either reopen or start a new one when ready 🤝